googletest/include/gtest/gtest-param-test.h

```diff
@@ -1419,14 +1419,14 @@ internal::CartesianProductHolder10<Generator1, Generator2, Generator3,
 // alphanumeric characters or underscore.
 
 # define INSTANTIATE_TEST_CASE_P(prefix, test_case_name, generator, ...) \
-  ::testing::internal::ParamGenerator<test_case_name::ParamType> \
+  static ::testing::internal::ParamGenerator<test_case_name::ParamType> \
```

Comment:
@davidben please dont edit this file firectly. Rather, change the corresponding pump file and run the pump script.

Reply:
Done. Interestingly, it doesn't look like anyone has run that script for a while. Some comments have been out of sync since 794ef030ebad671b699abdfd8e0474f93045be91. :-)